lib/perm/own-badge.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+function ownBadge (args, callback) {
+    var seneca = this;
+    var requestUser = args.user.id;
+    var badgeUser = args.params.badgeData.userId;
+    var validity = false;
```

Comment:
Why not just?
```
var validity = (requestUser === badgeUser);
```
or even forget about the validity var altogether and just go straight to
```
return callback(null, {'allowed': (requestUser === badgeUser)});
```